scripts/ssb_alert_manage

```diff
@@ -0,0 +1,89 @@
+##H Daemon for Alerting Tool (CERN SSB Status Board)
+##H Usage: ssb_alert_manage <Actions>
+##H
+##H Actions:
+##H   help        help manual
+##H   start       start the service
+##H   stop        stop the service
+##H   status      output service's status
+##H
+
+
+### ***
+#The following block of code was taken from
+#https://github.com/dmwm/deployment/blob/master/exitcodes/manage#L27
+
+ME=$(basename $(dirname $0))
+TOP=$(cd $(dirname $0)/../../.. && pwd)
+ROOT=$(cd $(dirname $0)/../.. && pwd)
+LOGDIR=$TOP/logs/$ME
+COLOR_OK="\\033[0;32m"
+COLOR_WARN="\\033[0;31m"
+COLOR_NORMAL="\\033[0;39m"
+
+#. $ROOT/apps/$ME/etc/profile.d/init.sh
+
+### ***
+
+# Actions Logic
+start()
+{
+    echo "Starting Alerting daemon."
+    nohup ssb_alert.sh $query $token </dev/null 2>&1 | rotatelogs $LOGDIR/ssb_alerting-%Y%m%d-`hostname -s`.log 86400 >/dev/null 2>&1 &
```

Comment:
what will happen if system does not have rotatelogs executable? Where $query and $token are defined? How to pass additional parameters to ssb_alert.sh?

You may pass query/token and other parameters via environment variables to manage script, but please make that they will have valid defaults. Then pass all parameters to ssb_alert.sh

The bottom line, please check that everything works and adjust manage script to work with and without rotatelogs. Check that $LOGDIR is actually present on a system and manage script has permission to write to it

Reply:
Valentin,

I have implemented checks for rotatelogs and $LOGDIR. Regarding arguements for ssh_alert.sh, I assumed enivornment variables only in the last commit as well, but all the arguements were not present (optional args). In latest commit this has been implemented - 
```
## ssb_alert.sh arguments from Environment Variable
QUERY=$query                                  ## monit influxDB query
TOKEN=$token                                  ## token file
URL=${url:-"https://cms-monitoring.cern.ch"}   ## alertmanager_url
INTERVAL=${interval:-1}                       ## time interval at which fetching and injecting data repeats
VERBOSE=${verbose:-0} 
```
Where query, token are required environment variables and url,interval,verbose are optional. Here they will have default values if not provided explicitly.